tests/cpp20_modules/foo.cpp

```diff
@@ -0,0 +1,25 @@
+module;
+
+#include <proxy/proxy_macros.h>
+
+#include <type_traits>
+#include <utility>
+
+export module foo;
+
+import proxy;
+
+extern "C++" {
+PRO_DEF_MEM_DISPATCH(MemGetFoo, GetFoo);
+PRO_DEF_FREE_DISPATCH(FreeGetBar, GetBar);
+PRO_DEF_FREE_AS_MEM_DISPATCH(FreeGetBaz, GetBaz);
+}
+
+export struct Foo
+    : pro::facade_builder ::add_convention<MemGetFoo, int() const>::build {};
```

Comment:
There should not be a white space between `facade_builder` and `::add_convention`